### DISCUSSION TITLE
Subscript and backticks for raw code in math lead to bug

### DISCUSSION THREAD

**Discussion by altosaar** [typst/typst#4608] [General]
### Description

WE are loving @typst at @onefact, and even taught a course using it! We made a template here: 

https://github.com/onefact/datathinking.org-report-template/

However:

<img width="578" alt="image" src="https://github.com/user-attachments/assets/86813574-dc3c-4555-8174-de4574d34e45">

Typst code to reproduce: 

```
In the above example, there would be two types of type embeddings, one for diagnosis codes— $E_`diag`$ and one for procedure codes.
```

### Reproduction URL

https://github.com/typst/typst/issues/new?assignees=&labels=bug&projects=&template=bug.yml&title=Write+a+short+and+descriptive+title%21

### Operating system

Web app, Windows, Linux, macOS

### Typst version

- [X] I am using the latest version of Typst

**Comment by Enivex** [comment]
This isn't a bug, the backtick syntax for raw simply isn't active in math mode. You don't need subscripts to exhibit this behavior.
You want #\`diag\`

**Reply by EpicEricEE** [reply]
Alternatively, if you want to use the monospaced glyphs of the math font, you might want to use `E_mono("diag")`

**Reply by Enivex** [reply]
> Alternatively, if you want to use the monospaced glyphs of the math font, you might want to use `E_mono("diag")`

Technically those aren't actually supposed to be used for text, just like everything else in the math font, but sure. I guess it's less problematic for monospace characters, since kerning isn't an issue.
